_posts/2021-07-16-release-0.9-future.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Release of v0.9 and the Future of wgpu
+---
+
+gfx-rs community's goal is to make graphics programming in Rust easy, fast, and reliable. Our current main projects are:
+
+  - [gfx-rs](https://github.com/gfx-rs/gfx) makes low-level GPU programming portable with low overhead. It's a single Vulkan-like Rust API with multiple backends that implement it: Direct3D 12/11, Metal, Vulkan, and even OpenGL ES.
+  - [naga](https://github.com/gfx-rs/naga) translates the shaders between languages, including WGSL. Also provides validation and processing facilities on the intermediate representation.
+  - [wgpu](https://github.com/gfx-rs/wgpu) is built on top of gfx-rs and [gpu-alloc](https://github.com/zakarumych/gpu-alloc)/[gpu-descriptor](https://github.com/zakarumych/gpu-descriptor). It provides safety, accessibility, and strong portability of applications.
+
+Following our release cadence every few months, we rolled out the 0.9 version through all of the gfx projects!
+See [gfx-rs changelog](https://github.com/gfx-rs/gfx/blob/master/CHANGELOG.md#hal-090-18-06-2021), [wgpu changelog](https://github.com/gfx-rs/wgpu/blob/master/CHANGELOG.md#v09-2021-06-18), and [naga changelog](https://github.com/gfx-rs/naga/blob/master/CHANGELOG.md#v05-2021-06-18) for the details.
+
+`naga` has matured signifgantly since the last release. `wgsl` parsing has imporved incredibly, targetting an up-to-date spec. `spirv` parsing support has had numerous bugs fixed, and `glsl` support is starting to take shape, though still in an alpha state.
```

Comment:
should this become a list?
>`naga` has matured signifgantly since the last release:
>  - `wgsl` ...
>  - `spirv` ..., but still has major open issues with control flow
>  - `glsl` ...
>  - general validation and correctness in other endpoints